guide/popular-topics/builders.md

```diff
@@ -0,0 +1,123 @@
+# Builders
+
+discord.js provides a `builders` package which contains a variety of utilities you can use when writing your Discord bot.
+To install the package, run `npm install @discordjs/builders` in your terminal.
+
+## Formatters
+
+Formatters are a set of utility functions which format input strings into the given format.
+
+### Basic Markdown
+
+The Formatters provide functions to format strings into all the different Markdown styles supported by Discord.
+
+```js
+const { bold, italic, strikethrough, underscore, spoiler, quote, blockQuote } = require('@discordjs/builders');
+const string = 'Hello!';
+
+const boldString = bold(string);
+const italicString = italic(string);
+const strikethroughString = strikethrough(string);
+const underscoreString = underscore(string);
+const spoilerString = spoiler(string);
+const quoteString = quote(string);
+const blockquoteString = blockQuote(string);
```

Comment:
```suggestion
const blockQuoteString = blockQuote(string);
```

Reply:
Blockquote is one word

Reply:
Oh, so is this a naming error in the builders package?